release/release_workflow.wdl

```diff
@@ -186,7 +186,49 @@ task versionPrep {
     }
 }
 
+task makeGithubRelease {
+    String githubToken
+    String organization
+    File cromwellJar
+    Int oldVersion
+    Int newVersion
+    
+    command <<<
+        set -e
```

Comment:
I just did and fixed it accordingly. I also tested the script on my Cromwell fork.